libs/domain/site/mode-selector/src/mode-selector.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { css } from 'lit';
+
+export const styles = css`
+  button {
+    color: var(--oryx-color-primary-300);
+  }
+
+  button:hover {
+    background-color: transparent;
+    box-shadow: none;
+    border-color: var(--oryx-color-primary-300);
+  }
+
+  button:focus-visible {
+    outline: solid 1px blue;
+  }
```

Comment:
I'd drop all styles except for maybe the color. It's not great, but it clearly shows that there's work left on the design system to make it look good.